Jenkinsfile.sonatype.deploy

```
@@ -13,6 +13,9 @@ def jira = [
 // Get jenkins plugin version
 def version = getVersionFromBuildName(env.releaseBuild_NAME)
 
+def segments = parseVersionString(getVersionFromBuildName(env.releaseBuild_NAME))
```

Can be

```suggestion
def segments = parseVersionString(version)
```

`version` coming from line 14?